src/lib/links.store.ts

```diff
@@ -0,0 +1,11 @@
+import { readable, type Readable } from 'svelte/store';
+
+export interface Links {
+  termsAndConditions?: string;
+}
+
+export let links: Readable<Links>;
```

Comment:
I implemented the store more like the style.store instead of locale.store, since it feels closer to this in nature - i.e. reading some extra props we define in the widget config, instead of the more advanced behaviour of locale.store.

This means that we don't export something similar to `partialStringsSchema` as the locale store does. So this differs slightly from the suggestion @cerebrl made in our chat on slack, but for me this is a simple implementation that works and is consistent with other stores. Having said that I welcome feedback on my approach during the review.

Reply:
Unfortunately, the `style.store` is incomplete. The `locale.store` is the best example as it's complete in its implementation. The difference is the use of Zod. Zod provides both build-time type safety, _plus_ runtime type feedback, which is critical to help developers catch silly misspellings and such. 

This is important for config and instantiation (public APIs) as it will eliminate those hard to catch bugs from improper object structure or typos in object keys. The convention is to use Zod for public APIs as devs may not always use TypeScript well or at all. We don't heavily use it internally, but this may change over time.

Reply:
Ok, this makes sense to me now. I've updated the links store to use Zod. Thanks.